ping/rust/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+edition = "2021"
+name = "testplan"
+version = "0.1.0"
+
+[features]
+v044 = ["libp2pv0440"]
+v045 = ["libp2pv0450"]
+v046 = ["libp2pv0460"]
+v047 = ["libp2pv0470"]
+
+[dependencies]
+async-std = {version = "1.10", features = ["attributes", "tokio1"]}
+env_logger = "0.9.0"
+if-addrs = "0.7.0"
+# Note: if this version doesn't match EXACTLY the testground one, I got errors.
+# Is it possible we get similar issues with libp2p dependencies?
```

Comment:
Yes, that is expected behavior. In case we ever hit the same issue with libp2p we can differentiate by version via the passed in libp2p feature flag.